docs/examples/robust_paper/finite_difference_eif/distributions.py

```diff
@@ -0,0 +1,34 @@
+from .abstractions import ModelWithMarginalDensity, FDModelFunctionalDensity
+from scipy.stats import multivariate_normal
+import pyro
+import pyro.distributions as dist
+
+
+class MultivariateNormalwDensity(ModelWithMarginalDensity):
```

Comment:
In a separate PR, could you add this to `MODEL` in `statics.py` as an element of `MultivariateNormalModel`.

Reply:
https://github.com/BasisResearch/chirho/pull/510/commits/719cff8d8d7d3da9932e9673593c3c23bdf5f14d